docs/src/1_Mechanics/04_Powers.yaml

```diff
@@ -954,6 +966,24 @@ Thwart:
     Prereq:
         Level: 2
 
+Infuse:
+    Category: Discard (limit one per character)
+    Description: You pour yourself into helping others.
+    Mechanic: Discard N cards to add +(N/2) HP or AP to the next Power you use that already grants HP or AP.
+    Type: Minor
+    XP: 0
+    Prereq:
+        Level: 2
```

Comment:
I like this new mechanic. Adding + to HP/AP to amplify an affect feels like a good use of Stamina.